lib/resources/r4/procedure.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Procedure
+field_name_base_url: https://hl7.org/fhir/r4/condition-definitions.html#Procedure
+fields:
+- name: code
+  required: 'Yes'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: A code to identify a specific procedure.
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.code
```

Comment:
This needs to go to R4

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/783dce5f317c0ee97b87117e79fb62dcbca23583#diff-80d4a0c17acadba0cd26a39c2cfeaf8dR23).

Reply:
+1